src/bin/rad-ci.rs

```diff
@@ -0,0 +1,5 @@
+use rad_ci::{run, Options, HELP};
+
+fn main() {
+    rad_terminal::args::run_command::<Options, _>(HELP, "ci", run);
```

Comment:
The string here is prepended to failure messages like this: `{string} failed: {error}`.

So let's set it to just "Command" if this tool can do multiple things, then it'll show `Command failed: {error}`.

Reply:
Are you sure about this? won't be too informative, also as of now, it's only doing install/uninstalls.